README.md

```diff
@@ -3831,6 +3920,31 @@ Tasks which only install some dependency but do not invoke any command start wit
 install_crate = { rustup_component_name = "rust-src" }
 ```
 
+<a name="error-codes"></a>
+## Error Codes
+
+<a name="e001"></a>
+### E001: Cycle Detected
```

Comment:
Maybe 'Environment Variables Cycle Detected' since other cycles are possible (task dependencies for example)